### PR TITLE
Netctl changes to support tenant commands

### PR DESCRIPTION
**Pull request by DivyaVavili**
Add tenant manipulation commands (create/delete/list) to netctl

vagrant@netplugin-node1:~$ netctl tenant create -p 100.1.0.0/16 -v 2000-2100 -x 12000-12100 tenant1
INFO[0000] Creating tenant: tenant1
vagrant@netplugin-node1:~$ netctl tenant list
Name     Subnet Pool     Subnet Len  vlans        vxlans
------   -----------     ----------  -----        ------
default  10.1.1.1/16     24          100-1100     1001-1100
tenant1  100.1.0.0/16    24          2000-2100    12000-12100
vagrant@netplugin-node1:~$ netctl tenant delete tenant1
INFO[0000] Deleting tenant tenant1
vagrant@netplugin-node1:~$ netctl tenant list
Name     Subnet Pool    Subnet Len  vlans       vxlans
------   -----------    ----------  -----       ------
default  10.1.1.1/16    24          100-1100    1001-1100

Arguments help:
vagrant@netplugin-node1:~$ netctl tenant create -h
NAME:
   netctl tenant create - Create a tenant

USAGE:
   netctl tenant create [command options] [tenant]

OPTIONS:
   --subnet-pool, -p            Subnet CIDR - REQUIRED
   --subnet-len, -l "24"        Subnet length
   --vlans, -v                  Vlan range - REQUIRED
   --vxlans, -x                 Vxlan range - REQUIRED

vagrant@netplugin-node1:~$ netctl tenant list -h
NAME:
   netctl tenant list - List tenants

USAGE:
   netctl tenant list

vagrant@netplugin-node1:~$ netctl tenant delete -h
NAME:
   netctl tenant delete - Delete a tenant

USAGE:
   netctl tenant delete [tenant]

### CODE REVIEW

File: netctl/netctl.go [hunk 1]
```diff
@@ -252,6 +252,71 @@ func listNetworks(ctx *cli.Context) {
...
+	argCheck(0, ctx)
+
+	list := getList(ctx, tenantURL(ctx))
+
+	if ctx.Bool("json") {
+		dumpList(ctx, list)
+	} else {
+		writer := tabwriter.NewWriter(os.Stdout, 0, 4, 2, ' ', 0)
+		defer writer.Flush()
+		writer.Write([]byte("Name  \tSubnet Pool\t\tSubnet Len\tvlans\t\tvxlans\n"))
```
Comment (erikh) [inline netctl/netctl.go]: The tabwriter will automatically indent these properly, so you should not need to space the `Name` out.

**Comment by erikh** [comment]
netctl LGTM other than that one nit

**Comment by shaleman** [comment]
@DivyaVavili: Looks like this PR has both netctl changes and other svcplugin cleanups.
Do you want to resubmit a combined PR or submit only netctl changes as a separate PR

**Comment by DivyaVavili** [comment]
I will have the netctl changes in a separate PR.

**Comment by DivyaVavili** [comment]
Raised another PR (PR #201 )with this change included.

**Pull request closed without merging**